sdk/communication/communication-common/src/identifierModelSerializer.ts

```diff
@@ -34,9 +34,9 @@ export interface SerializedCommunicationIdentifier {
    */
   microsoftTeamsUser?: SerializedMicrosoftTeamsUserIdentifier;
   /**
-   * The Microsoft bot.
+   * The Microsoft Teams App.
    */
-  microsoftBot?: SerializedMicrosoftBotIdentifier;
+  microsoftTeamsApp?: SerializedMicrosoftTeamsAppIdentifier;
```

Comment:
Hi @DominikMe  @AikoBB Chat Signaling Client also has Communication Identifier model in API which is not dependent on common package. Could you please also create the work item for our team to update the model in signaling client? 

@DominikMe Do you suggest to make @azure/communictaion-signaling consume common package directly to avoid the model inconsistency？

Reply:
Hi @LuChen-Microsoft! I mentioned in the user story description to ping your team.

Reply:
@LuChen-Microsoft Let's chat more about adding the dependency to common. I'm trying to remember why we had avoided to add it in the past